schema_based_on_diagram.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE patients (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR,
+    date_of_birth DATE
+);
+
+CREATE TABLE invoices (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    total_amount DECIMAL,
+    generated_at TIMESTAMP,
+    payed_at TIMESTAMP,
+    medical_history_id INTEGER REFERENCES medical_histories(id)
+);
+
+CREATE TABLE medical_histories (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patient_id INTEGER REFERENCES patients(id),
+    status VARCHAR
+);
+
+CREATE TABLE invoice_items (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    unit_price DECIMAL,
+    quantity INTEGER,
+    total_price DECIMAL,
+    invoice_id INTEGER REFERENCES invoices(id),
+    treatment_id INTEGER REFERENCES treatments(id)
+);
+
+CREATE TABLE treatments (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    type VARCHAR,
+    name VARCHAR
+);
+
+CREATE TABLE medical_histories_treatments (
+    medical_history_id INTEGER REFERENCES medical_histories(id);
```

Comment:
- [ ] Please make use to fix this syntax error, you used `;` instead of `,`. please fix it in order to make your schema syntax correct :+1: